### DISCUSSION TITLE
Welcome new committer: Zhang Li Bin

### DISCUSSION THREAD

**Discussion by seeflood** [mosn/layotto#352] [Announcements]
Hi Community,

On behalf of the Layotto PMC, I am pleased to announce that Zhang Li Bin a.k.a ZLBer, has been voted in as Layotto committer.
![image](https://user-images.githubusercontent.com/26001097/144787404-a628ef88-ead5-4f9c-b28a-84747a3a28aa.png)

Zhang Li Bin has been making great contributions to Layotto. Please join me to say congratulations to him!

@ZLBer , would you please briefly introduce yourself to the community?

Best Regards.

**Comment by ZLBer** [comment]
hi everyone, I am Zhang Libin.  A student who will graduate soon(bad news), like coding, like tech, like algorithmic problem. Thank the community for the opportunity. I learned a lot form the commit and review.  Thanks again.

**Comment by zhenjunMa** [comment]
welcome!

**Comment by nobodyiam** [comment]
Congratulations! 🎉🎉🎉

**Comment by wenxuwan** [comment]
welcome, thanks for your contribution. 🎉🎉🎉

**Comment by MoonShining** [comment]
🎉🎉🎉
